pages/payroll.tsx

```diff
@@ -3,6 +3,10 @@ import { useEffect, useState } from 'react'
 import MultiRangeSlider from '../components/multi-range-slider';
 import { eachDayOfInterval, subYears, addMonths, startOfToday, } from 'date-fns'
 import { formatDate, formatCurrency, parseDate } from '../util/format';
+import ToggleButton from '@mui/material/ToggleButton';
```

Comment:
I am being picky, but you are technical importing a different design system than what is already being used, so it will infer the design styles that Material UI has set as defaults, which does not necessarily align with how the current app has been designed. Is it possible to not use this ToggleButton component? 

It looks like you can use the Tab HeadlessUI component, then just style with Tailwindcss. https://headlessui.com/react/tabs